.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+# If this file is renamed, the incrementing run attempt number will be reset.
+
+name: CI
+
+on:
+  push:
+    branches: [ "dev", "main" ]
+  pull_request:
+    branches: [ "dev", "main" ]
+
+env:
+  CI_BUILD_NUMBER_BASE: ${{ github.run_number }}
+  CI_TARGET_BRANCH: ${{ github.head_ref || github.ref_name }}
+
+jobs:
+  build:
+
+    runs-on: windows-latest
```

Comment:
Nothing Windows-specific, could run on Linux?

Reply:
I only left this on Windows because it uses PowerShell and wasn't sure if that's available on the default Linux runners. I'll give it a go and see what happens

Reply:
:+1: `pwsh` is there